static/templates/widgets/todo_widget_tasks.hbs

```diff
@@ -3,14 +3,14 @@
 <li>
     <button class="task" data-key="{{ key }}">
     </button>
-    <span class="task">{{ task }}</span>
+    <span class="task"><b>{{ task }}</b></span>
```

Comment:
We generally use the modern `strong` and `em` tags, not b/i.

Reply:
Ah, I'll change that. Should I add a new commit and push or squash all the commits abd push? As the documentation reads that solving of a particular issue should belong to a single commit.

Reply:
@majordwarf You should add it to the 1st commit. You can do it in a number of ways. Rebasing or `git commit --amend` can both help with that.